pymc3/pytestdebug.log

```diff
@@ -0,0 +1,1265 @@
+versions pytest-5.0.1, py-1.8.0, python-3.7.4.final.0
```

Comment:
This log file shouldn't be here. Maybe you could add a rule to .gitignore

Reply:
I will remove it from the history and add a rule to .gitignore.

Reply:
Removed it from the history.  There's a rule for it in .gitignore.   I must have accidentally committed it in the blob immediately before the blob where I committed the rule in .gitignore.